docs/schema.graphql

```diff
@@ -91,19 +91,34 @@ type ContentType implements Node {
     orderBy: DepositOrder
 
     """
-    List only those elements that have a timestamp earlier than this given timestamp.
+    List only those deposits that have a creation timestamp earlier than this given timestamp.
     """
-    earlierThan: DateTime
+    createdEarlierThan: DateTime
 
     """
-    List only those elements that have a timestamp later than this given timestamp.
+    List only those deposits that have a creation timestamp later than this given timestamp.
     """
-    laterThan: DateTime
+    createdLaterThan: DateTime
 
     """
-    List only those elements that have a timestamp equal to the given timestamp.
+    List only those deposits that have a creation timestamp equal to the given timestamp.
     """
-    atTimestamp: DateTime
+    createdAtTimestamp: DateTime
+
+    """
+    List only those deposits that have a last modified timestamp earlier than this given timestamp.
+    """
+    lastModifiedEarlierThan: DateTime
+
+    """
+    List only those deposits that have a last modified timestamp later than this given timestamp.
+    """
+    lastModifiedLaterThan: DateTime
+
+    """
+    List only those deposits that have a last modified timestamp equal to the given timestamp.
```

Comment:
I wasn't sure, so I did some experimenting:

value in database | accepted search queries
----- | -------------------------
`2019-03-03 02:02:00+01` | `2019-03-03T02:02`, `2019-03-03T02:02+01`, `2019-03-03T02:02+01:00`, `2019-03-03T02:02:00+01:00`, `2019-03-03T02:02:00.000+01:00`
`2020-05-12 14:37:07.073+02` | `2020-05-12T14:37:07.073`, `2020-05-12T14:37:07.073+02:00`, `2020-05-12T12:37:07.073Z`

I understand your concern about millisecond precision, but that is how it is implemented. A use case for this might be that you see something being logged that you want to find based on that timestamp. Otherwise `lastModifiedAtTimestamp` might not be of much use, however I need `lastModifiedEarlierThan` and `lastModifiedLaterThan` (equivalent to `<` and `>`), so it made sense to have `lastModifiedAtTimestamp` (equivalent to `==`) as well. Same goes for `createdAtTimestamp`.